src/sidebar/store/modules/defaults.js

```diff
@@ -0,0 +1,47 @@
+import * as util from '../util';
+
+/**
+ * A store module pertaining to client-side defaults for (browser) users.
+ */
+
+function init(settings, persistedDefaults) {
+  // Initialize state from any available persisted default values
+  return {
+    annotationPrivacy: persistedDefaults.annotationPrivacy || 'shared',
+  };
+}
+
+const update = {
+  SET_DEFAULT: function(state, action) {
+    return { [action.defaultKey]: action.value };
+  },
+};
+
+const actions = util.actionTypes(update);
+
+function setDefault(defaultKey, value) {
```

Comment:
Right now, there is nothing guarding setting or accessing an unknown default; we could add that in at any point.